package.json

```diff
@@ -52,7 +52,7 @@
     "jed": "^1.1.1",
     "lodash": "^4.17.5",
     "patternfly": "^3.41.1",
-    "patternfly-react": "^2.1.0",
+    "patternfly-react": "2.5.1",
```

Comment:
I'm usually not a fan of such strict pinning. Since Foreman also has this dependency they must be in exact sync. Should we pin to `~2.5.1` since it's a bit more loose?

Reply:
I afraid it won't actually work.
Foreman will receive `2.5.1` and Katello will receive `2.5.2` so we will get a duplication in our `vendor.js`.

I do think we should consider moving packages like patternfly-react to `peerDependencies`.

Reply:
I agree it should actually be peerDependencies. The foreman should provide a package with dependencies that plugins can depend on.

Reply:
@ekohl can we unblock nightly first? I would be happy to hear your proposal to fixing this long term.